index.js

```diff
@@ -26,8 +26,16 @@ app.use(methodOverride(function (req, res) {
 
 app.use('/admin', adminRouter);
 
+app.post('/comments', (req, res) => {
+  db.Comment.create(req.body).then((comment) => {
+    return comment.getPost().then((post) => {
+      res.redirect('/' + post.slug);
+    });
+  });
+});
+
 app.get('/', (req, res) => {
-  db.Post.findAll().then((blogPosts) => {
+  db.Post.findAll({ order: [['createdAt', 'DESC']] }).then((blogPosts) => {
```

Comment:
it will find all posts order by their descending createdAt order. It will generate a SQL query with ORDER BY. We use order: [order1, order2, ...] syntax since this seems to be the supported way to declare orderings in sequelize, it lets us create more complex orders(get something with two columns ordered at the same time) in case if we need to.